_data/compatibility/xapo.yaml

```diff
@@ -0,0 +1,82 @@
+---
+name: Xapo
+internal_url: /en/compatibility/xapo
+logo: /img/compatibility/xapo/xapo.png
+rbf:
+  tested:
+    date: "2018-11-05"
+    platforms:
+      - iOS
+    version: "4.6.1"
+  features:
+    receive:
+      notification: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "true"
+      shows_original_version: "true"
+    send:
+      signals_bip125: "false"
+      list: "false"
+      details: "false"
+      shows_replaced_version: "na"
+      shows_original_version: "na"
+  examples:
+    - image: /img/compatibility/xapo/send-screen.png
+      caption: >
+        Sending RBF Transaction - Default wallet send screen
+    - image: /img/compatibility/xapo/send-miner-fee-options.png
+      caption: >
+        Sending RBF Transaction - Send transaction, choose miner fees. No RBF options.
+    - image: /img/compatibility/xapo/tranasction-details-sent.png
+      caption: >
+        Bumping RBF Transaction - Transaction not sent with RBF signal. No bumping possible.
+    - image: /img/compatibility/xapo/notification-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Email notification. No RBF warning.
+    - image: /img/compatibility/xapo/transaction-list-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Incoming RBF transaction in list. No label for RBF.
+    - image: /img/compatibility/xapo/transaction-details-incoming-rbf.png
+      caption: >
+        Receiving RBF Transaction - Incoming RBF transaction details. No RBF flag.
+    - image: /img/compatibility/xapo/transaction-list-incoming-replacement.png
+      caption: >
+        Receiving Bumped RBF Transaction - Incoming bumped transaction. Both transactions show. Balance credited twice.
+    - image: /img/compatibility/xapo/dashboard-incoming-replacement.png
+      caption: >
+        Receiving Bumped RBF Transaction - Dashboard shows 2 incoming transaction. Balance includes both transactions.
+    - image: /img/compatibility/xapo/notification-incoming-replacement.png
+      caption: >
+        Receiving Bumped RBF Transaction - Additional email notification for bumped transaction.
+    - image: /img/compatibility/xapo/transaction-list-replacement-confirmed.png
+      caption: >
+        Receiving Bumped RBF Transaction - Even after bumped transaction has 100 confirms, original transaction stays pending.
+segwit:
+  tested:
+    date: "2019-07-27"
+    platforms:
+      - iOS
+    version: "5.0.3"
+  features:
+    receive:
+      p2sh_wrapped: "false"
```

Comment:
This seems inconsistent. It says xapo does not support receiving the wrapped-segwit, but its default receiving address is wrapped-segwit.

Reply:
Inconsistent and just WRONG. Corrected.